src/tests/cucumber/features/solver-features/hybrid_studies.feature

```diff
@@ -58,4 +58,18 @@ Feature: hybrid (simulator+modeler) studies
     # for now, modeler costs does not figure in system cost txt
     And the annual system cost is 17640000.0
     And in area "AREA", during year 1, loss of load lasts 0 hours
-    And in area "AREA", during year 1, hourly production of "base" is always equal to 3000 MWh
\ No newline at end of file
+    And in area "AREA", during year 1, hourly production of "base" is always equal to 3000 MWh
+
+  @fast @short
+  Scenario: Legacy node with one legacy load (up to 5952 MW) and wind, and one generator component (168h simplex)
+    # copy of test 3_6_1, both weeks are clones
+    # except max_p of the generator component is 6200 in first week, 5900 in second
+    Given the solver study path is "Antares_Simulator_Tests_NR/hybrid/3_6_4"
+    When I run antares simulator
+    Then the simulation succeeds
+    And the simulation takes less than 5 seconds
+    # for now, modeler costs does not figure in system cost txt
+    And the annual system cost is 520000
+    And in area "AREA", during year 1, week 1, loss of load lasts 0 hours
+    And in area "AREA", during year 1, week 2, loss of load lasts 1 hours
```

Comment:
In my opinion you should compare the UNSP ENRG (volume of unsupplied energy), not the LOLD. The LOLD is too sensitive to equivalent solutions.

Reply:
it can't depend on equivalent solutions here, because load > Pmax, and there is no storage

Reply:
For this specific example, OK. Generally speaking, prefer UNSP ENRG